app/javascript/components/miq-structured-list/index.jsx

```diff
@@ -109,11 +109,26 @@ const MiqStructuredList = ({
   /** Function to render textarea / checkbox / react components */
   const renderInputContent = ({ value }) => <MiqStructuredListInputs value={value} action={(data) => onClick(data)} />;
 
+  /** Function to include content for mode that contains miq summary*/
+  const renderContentsForMiqSummary = (row, content) => {
```

Comment:
Can we change the name of the function to something else more specific to what the part does?

Reply:
@jeffibm  used **renderLinkWithHrefOrOnclickForMiqSummary** as name for function which is more descriptive and also updated the comment line above on what exactly the function does. please check